rust-crates/symblib/src/symbconv/go.rs

```diff
@@ -67,18 +67,45 @@ fn extract_ranges(obj: &objfile::Reader<'_>, visitor: super::RangeVisitor<'_>) -
     stats.is_go_binary = true;
 
     let mut func_iter = go.funcs()?;
-    while let Some(func) = func_iter.next()? {
+    'outer: while let Some(func) = func_iter.next()? {
```

Comment:
Just a personal preference: Instead of a label to break the loop, it might be better to extract the functionality into its own function.

Reply:
That's my preference, too. I struggled with owner ship, had to read up a bit more.